### PR TITLE
Add option to delete saved dashboards and recent hosts to the settings screen

### PR DESCRIPTION
**Pull request by basyskom-jvoe**

### CODE REVIEW

File: src/backend.cpp [hunk 1]
```diff
@@ -705,6 +728,25 @@ void BackEnd::trustCertificate()
     connectToEndpoint();
 }
 
+void BackEnd::removeRecentConnection(const QString &name)
+{
+    if (mLastServerHosts.contains(name)) {
+        mLastServerHosts.removeAll(name);
+
+        QSettings settings;
```
Comment (khaexy) [inline src/backend.cpp:736]: The same source code is used in method saveServerHost(). It is best to write a separate method.

File: src/qml/SettingsView.qml [hunk 2]
```diff
@@ -208,6 +208,196 @@ Rectangle {
...
+                        delegate: Rectangle {
+                            id: variableDashboardsListViewDelegate
+
+                            required property int index
+                            required property string display
+
+                            radius: 5
+                            width: variableDashboardsListView.width
+                            implicitHeight: childrenRect.height
+                            color: view.theme.backgroundSelected
```
Comment (khaexy) [inline src/qml/SettingsView.qml:258]: set color to "transparent"

File: src/qml/SettingsView.qml [hunk 3]
```diff
@@ -208,6 +208,196 @@ Rectangle {
...
+
+                                Text {
+                                    Layout.fillWidth: true
+                                    Layout.rightMargin: 5
+                                    Layout.leftMargin: 5
+                                    font {
+                                        pointSize: 11
+                                    }
+                                    text: display
+                                    color: view.theme.textColor
```
Comment (khaexy) [inline src/qml/SettingsView.qml:280]: set color to "transparent"

File: src/qml/SettingsView.qml [hunk 4]
```diff
@@ -208,6 +208,196 @@ Rectangle {
...
+                            required property int index
+                            required property string display
+
+                            radius: 5
+                            width: variableDashboardsListView.width
+                            implicitHeight: childrenRect.height
+                            color: view.theme.backgroundSelected
+                            clip: true
+
+                            MouseArea {
```
Comment (khaexy) [inline src/qml/SettingsView.qml:261]: delete MouseArea

File: src/qml/SettingsView.qml [hunk 5]
```diff
@@ -208,6 +208,196 @@ Rectangle {
...
+                            required property int index
+                            required property string modelData
+
+                            radius: 5
+                            width: recentConnectionsListView.width
+                            implicitHeight: childrenRect.height
+                            color: view.theme.backgroundSelected
+                            clip: true
+
+                            MouseArea {
```
Comment (khaexy) [inline src/qml/SettingsView.qml:356]: delete MouseArea

**Review by khaexy — CHANGES_REQUESTED** [review]

File: src/qml/SettingsView.qml (hunk 2 above)
Comment (basyskom-jvoe) [reply src/qml/SettingsView.qml:258]: Done.

File: src/qml/SettingsView.qml (hunk 3 above)
Comment (basyskom-jvoe) [reply src/qml/SettingsView.qml:280]: Done.

File: src/qml/SettingsView.qml (hunk 4 above)
Comment (basyskom-jvoe) [reply src/qml/SettingsView.qml:261]: Done.

File: src/qml/SettingsView.qml (hunk 5 above)
Comment (basyskom-jvoe) [reply src/qml/SettingsView.qml:356]: Done.

File: src/backend.cpp (hunk 1 above)
Comment (basyskom-jvoe) [reply src/backend.cpp]: Done.

**Review by khaexy — APPROVED** [review]

**Pull request merged**